wiki/People/The_Team/Nomination_Assessment_Team/Evaluations/en.md

```diff
@@ -1,50 +1,24 @@
-
-Once three NAT members have submitted their individual evaluations, the application will automatically move to the group phase on the NAT website.
-
-### Group phase
-
-The group phase is given a deadline of 14 days after the applicant sent in their application. During the group phase, NAT members can read each other's evaluations and discuss further as needed to come to a final consensus. Once a consensus is reached, they will put the result on the NAT website and write the final evaluation feedback for the applicant. 
-
-In the case of [failed applications](/wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator#failed-applications), the NAT will also determine the length of the user's cooldown before they may apply again. The standard cooldown is 90 days from the date of the application, but the duration may be reduced for applicants with relatively few issues they need to work on before becoming a Beatmap Nominator. When this happens, the conditions for reduced cooldowns will be indicated in the feedback sent to the applicants.
-
-Once consensus is reached and feedback is written, NAT members will review the feedback to polish feedback accuracy and clarity for the applicant. They then send the feedback to the applicant, after which the evaluation will be archived for future reference. The feedback itself will delivered by the [mappersguild bot](https://osu.ppy.sh/users/23648635) as an osu! chat message.
-
 ## Current Beatmap Nominator evaluations
 
-Every BN is reviewed regularly by members of the NAT. 
+Every BN is reviewed regularly by members of the NAT.
 
 For Nominators on [probation](/wiki/People/The_Team/Beatmap_Nominators#probationary-beatmap-nominators), or that have been issued an [activity](/wiki/People/The_Team/Beatmap_Nominators/Rules#activity) warning, their next evaluation will be scheduled for roughly 1 month after being placed on probation or receiving their warning.
 
 For full Beatmap Nominators without recent warnings, their evaluations will be scheduled to occur every 3 months.
 
-The duration between evaluations is automatically scheduled by the [NAT website](https://bn.mappersguild.com/). However, if there is a substantial cause for concern about a BN's performance at a sooner date, NAT members can also manually schedule an evaluation to happen sooner than normal.
+The duration between evaluations is automatically scheduled by the [BN website](https://bn.mappersguild.com/). However, if there is a substantial cause for concern about a BN's performance at a sooner date, NAT members can also manually schedule an evaluation to happen sooner than normal.
 
 Just like applications, current BN evaluations are randomly assigned 3 NAT members and will be announced in the respective mode's NAT Discord channels.
```

Comment:
"becoming a beatmap nominator" page mentions 2 to 4

Reply:
I don't think we assign 4 currently, 3 at most, and 2 for the smaller gamemodes, though for the times when we invite BNs to evaluate the total number of evaluators can go up to 6.